ui/app/AppLayouts/Wallet/WalletLayout.qml

```diff
@@ -331,7 +330,7 @@ Item {
             }
             onLaunchBridgeModal: {
                 root.sendModalPopup.preSelectedSendType = Constants.SendType.Bridge
-                root.sendModalPopup.preSelectedRecipient = root.sendModalPopup.preSelectedAccount.address
+                root.sendModalPopup.preSelectedRecipient = root.sendModalPopup.preSelectedAccount
```

Comment:
This fixes launching the `BridgeModal`; it was aborting on this error:

```
WRN 2024-07-12 18:11:43.643+02:00 qt warning                                 topics="qt" tid=3453434 text="TypeError: Cannot read property 'address' of undefined" file=qrc:/app/AppLayouts/Wallet/WalletLayout.qml:333 category=default
```

The `preSelectedRecipient` is a `var` property and expects the whole object, not the address